scripts/test.sh

```diff
@@ -11,6 +11,9 @@ for f in $( git diff-tree $TRAVIS_COMMIT --name-status -r | grep php | grep -v "
 
 ./scripts/lint.sh
 
+export TERMINUS_CACHE_DIR="$HOME/.terminus/testcache"
+export TERMINUS_TOKENS_DIR="$HOME/.terminus/testcache/tokens"
```

Comment:
It now seems that when I run my unit tests that they are targeting terminus.pantheon.io instead of onebox, which they are fixtured for. Is it doing the same for you? If so, it's lost the TERMINUS_HOST setting somewhere.